contracts/core/RebalancingSetTokenFactory.sol

```diff
@@ -99,6 +99,9 @@ contract RebalancingSetTokenFactory {
         // Expecting caller to be Core
         require(msg.sender == core, "ONLY_CORE_CAN_CREATE_REBAL_SET");
 
+        // Ensure components and units length are 1
+        require(_components.length == 1 && _units.length == 1, "ARRAY_LENGTHS_MUST_BE_ONE");
```

Comment:
Can you make this message uniquely identifiable but < 32 characters? I recommend we change these around to something like: `RebalancingFactory#Const.ArrayLength` @bweick